setup.py

```diff
@@ -119,6 +127,53 @@ def run(self):
 cmdclass['test'] = NoopTestCommand
 cmdclass['build_ext'] = BuildExtraLibraries
 
+
+def _download_jquery_to(dest):
+    # Note: When bumping the jquery-ui version, also update the versions in
+    # single_figure.html and all_figures.html.
+    url = "https://jqueryui.com/resources/download/jquery-ui-1.12.1.zip"
+    if not os.path.exists(os.path.join(dest, "jquery-ui-1.12.1")):
```

Comment:
Better not to hard-code the subdirectory. This can easily go wrong in a future update.
~~~
targetdir = Path(dest) / Path(url.split('/')[-1]).stem
if not targetdir.exists():
~~~

Reply:
I think forgetting to update the HTML template is a much worse risk.  Plus see below re: py2 compatibility.